eng/tools/generator/cmd/v2/release/releaseCmd.go

```diff
@@ -111,6 +143,109 @@ func (c *commandContext) execute(sdkRepoParam, specRepoParam string) error {
 		return err
 	}
 
+	if c.flags.ReleaseRequest != "" {
```

Comment:
I think the post action for release request should be put into another help function. We can make the execution process more clearly for two kinds of input: rp or request.